tests/utils.js

```diff
@@ -132,3 +133,46 @@ export function dispatchMouseDownMoveOrUpEvent(element, type, clientX, clientY,
         }),
     );
 }
+
+export function getEventObjectBy3DPosition(position, camera) {
```

Comment:
@aslipchenk Let's add docstrings to these functions, please, with parameter type definitions

Reply:
Ok, since that PR is already closed I created a new one with fixes -> https://github.com/Exabyte-io/wave.js/pull/97